### ISSUE TITLE
[BUG][a11y] Voiceover doesn't work properly on SfSelect

### ISSUE THREAD

**Issue by Qrzy** [DivanteLtd/storefront-ui#1068]
As reported here: https://github.com/DivanteLtd/next/issues/390
Voiceover service is not able to read out options for SfSelect.
I think this component needs some lifting in the a11y topic.

P.S. Maybe this article would be helpful? This lady has good knowledge in a11y area, AFAIK.
https://dev.to/emmabostian/creating-a-custom-accessible-drop-down-3gmo
